CDCgov/prime-reportstream
# enable flywayClean for resetDB gradle task

This PR enables flywayClean within the `resetDB` gradle task. A recent flyway update defaulted flywayClean to being disabled. 

To test:

1. package
2. check to see if values exist in a db table
   - ex `select * from lookup_table_version`
   - if tables are empty run `./gradlew reloadTables`
4. run `./gradlew resetDB`
5. check that the table is empty
